src/SliderButton.js

```diff
@@ -1,74 +1,83 @@
 import React, { Component, PropTypes } from 'react';
+import { Button } from 'devui';
 
 export default class SliderButton extends Component {
   static propTypes = {
     theme: PropTypes.object,
-    replaySpeed: PropTypes.string,
     type: PropTypes.string,
+    disabled: PropTypes.bool,
     onClick: PropTypes.func
   }
 
-  iconStyle(theme) {
+  shouldComponentUpdate(nextProps) {
+    return this.props.type !== nextProps.type || this.props.disabled !== nextProps.disabled;
```

Comment:
I just tried it with the example, it actually didn't updated if I changed speed, so when I change to `Live` from `1x` (also use https://github.com/calesce/redux-slider-monitor/pull/26#discussion_r106467139 fix), it still use `startReplay` not `startRealtimeReplay`.

I agreed for use `PureComponent`. 👍